services/ai-navigator-cluster-info-agent/0.1.2/defaults/cm.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: ai-navigator-cluster-info-agent-0.1.1-d2iq-defaults
```

Comment:
`ai-navigator-cluster-info-agent` is a separate chart and we don't need to update it: https://github.com/mesosphere/ai-navigator-cluster-info-agent